javascript/javascript3/week3/repos.js

```diff
@@ -0,0 +1,51 @@
+const repoUsers = ['Jul-S', 'Sarulathaanbu', 'teldinesh']
+
+class Repo {
+    constructor(name, url, owner) {
+        this.name = name
+        this.url = url
+        this.owner = owner
+    }
+}
```

Comment:
This class is not used anywhere

Reply:
i just want to try with class ... but i felt not necessary